jsk_robot_common/jsk_robot_startup/msg/RoomLight.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+bool light_on
+float32 luminance
```

Comment:
use `senseor_msgs/Illuminance luminance` instead of `float32 luminance` http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Illuminance.html
I would prefer this node publish illuminance and it is client's job to judge if the light is on or off, but that is just my preference and make new message is ok.

Reply:
In the definition, illuminance should be `Lux` unit, but we calculate this relative luminance from RGB image. so I think we cannot use the message in order to avoid misunderstandings.
Otherwise, we can make `RelativeLuminance.msg`